bootstrap/exherbo.sh

```diff
@@ -0,0 +1 @@
+_exherbo_common.sh
```

Comment:
I followed the example in the mentioned pull request from https://github.com/letsencrypt/letsencrypt/pull/1221 and e.g. fedora.sh is also currently unused. Just tell me which one you prefer and I'll update my pull request accordingly if required.

1. leave it as it is
2. leave the symlink and refer to it in install-deps.sh/letsencrypt-auto instead of _exherbo_common.sh
3. rename _exherbo_common.sh to exherbo.sh and refer to that in install-deps.sh/letsencrypt-auto
4. just delete the redundant symlink